src/tests/infraestructure/routes/test-routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { TestController } from '../controller/TestController';
+
+const router = Router();
+const controller = new TestController();
+
+router.get('/', (req, res) => controller.getTests(req, res));
+
+router.get('/:id', (req, res) => controller.getTestById(req, res));
+
+router.post('/:id/start', (req, res) => controller.startTest(req, res));
```

Comment:
Route handlers are registered without authentication or validation middleware. Critical endpoints (start, questions, submit) should require authentication and request validation.

**Sugerencia:**
```typescript
Attach auth and validation middleware. Example:
router.post('/:id/start', authMiddleware, validateStart, (req, res) => controller.startTest(req, res));
Also validate :id is a valid UUID and validate request bodies. Priority: critical (security / input validation)
```